### PR TITLE
switch to augur curate normalize-strings

### PR DESCRIPTION
**Pull request by j23414**
### Description of proposed changes

Minor edit to move toward using the `augur curate` command where possible. 

https://docs.nextstrain.org/projects/augur/en/stable/usage/cli/curate/normalize-strings.html

### Testing

- [ ] Checks pass

### CODE REVIEW

**Comment by j23414** [comment]
Just checking in if there were any questions before merging this? Happy to chat or feel free to DM me.

File: ingest/workflow/snakemake_rules/transform.smk [hunk 1]
```diff
@@ -67,7 +67,7 @@ rule transform:
         (cat {input.sequences_ndjson} \
             | ./bin/transform-field-names \
                 --field-map {params.field_map} \
-            | ./bin/transform-string-fields --normalize \
```
Comment (victorlin) [inline ingest/workflow/snakemake_rules/transform.smk:70]: Suggestion: delete `transform-string-fields` if it is no longer used anywhere else.

**Review by victorlin — COMMENTED** [review]
This seems like a good change to me. Have you tested that it works, or is there another similar change in a different repo that can be referenced to back this change?

**Comment by j23414** [comment]
Good call @victorlin , I tested it in dengue (diff "data/metadata.tsv" of original compared to "augur curate" versions) but should have done the same test for rsv.

I'm waiting for my test to complete:

```
git clone https://github.com/nextstrain/rsv.git
cd rsv
nextstrain build ingest data/metadata.tsv
```

which is currently at:

https://github.com/nextstrain/rsv/blob/d94d23d07394ba2f89583bd30330a4ed883271c3/ingest/workflow/snakemake_rules/fetch_sequences.smk#L16-L32

I'll still need to another run with the modification and then I'll post when the comparison is done.

**Comment by j23414** [comment]
Done! And the two metadata.tsv's matched

File: ingest/workflow/snakemake_rules/transform.smk (hunk 1 above)
Comment (victorlin) [reply ingest/workflow/snakemake_rules/transform.smk:70]: I see, in that case a modified suggestion: remove what implements `--normalize` in `transform-string-fields`.

**Review by victorlin — APPROVED** [review]
Looks good, I didn't test anything – trusting your test results posted above.

File: ingest/workflow/snakemake_rules/transform.smk (hunk 1 above)
Comment (victorlin) [reply ingest/workflow/snakemake_rules/transform.smk:70]: Thanks, I see 2bc7409b9ab398e63aa340ee5b813efe566519e8. Would be nice to squash the two commits as one, so it's obvious why that change was made.

**Pull request merged**